theme6/source/stylesheets/main.css

```diff
@@ -0,0 +1,280 @@
+.featured-kits {
+  font-family: 'Oswald', sans-serif;
+}
+
+.big-banner {
+  background-image: url("../images/banner1_mobile.png");
+  background-size: 120%;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.big-banner-content {
+  background: linear-gradient(0deg,
+    rgba(251,239,227),
+    rgba(251,239,227, 1) 20%,
+    rgba(251,239,227, 0.2) 50%,
+    rgba(251,239,227, 0.1));
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 320px;
+  text-align: center;
+}
+
+.big-banner-content h1 {
+  margin-bottom: 0;
+  font-size: 20px;
+}
+
+.big-banner-content p {
+  font-family: 'Lato', sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+.big-banner-content button {
+  font-family: 'Oswald', sans-serif;
+  border-radius: 0;
+  border-style: none;
+  background-color: #4A90E2;
+  margin-bottom: 12px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
+
+.big-banner-content button h1 {
+  margin: 0;
+  padding: 10px 0 10px 0;
+  color: #FFFFFF;
+}
+
+.small-banner {
+  display: flex;
+  background-image: url("../images/banner2.png");
+  box-shadow: inset 0 0 0 1000px rgba(245, 166, 35,0.8);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 20px;
+  color: #FFFFFF;
+}
+
+.small-banner h1 {
+  margin: 0 0;
+  font-size: 45px;
+}
+
+.small-banner h3 {
+  margin: 0;
+}
+
+.mid-banner-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 300px;
+  text-align: center;
+}
+
+.mid-banner-saylor {
+  background-image: url("../images/mid_banner1.png");
+  background-size: 130%;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.mid-banner-archeologist {
+  background-image: url("../images/mid_banner2.png");
+  background-size: 130%;
```

Comment:
Remember percentages, should almost never be used. An alternative to this is to use padding or positioning the image in your container.